Code Review Assignment_JrSDET .java

```diff
@@ -12,7 +12,7 @@
 import org.openqa.selenium.interactions.Actions;
 
 
-
+//code review
 public class Testcase101 {

```

Comment:
Two Main classes (Testcase101 & PagesforAutomationAssignment) are in this file.These should be in separate files to follow to Java conventions.